### ISSUE TITLE
Tile flip interaction doesn't work well for screen readers

### ISSUE THREAD

**Issue by dougmacknz** [cultureamp/kaizen-design-system#1764]
This popped up in https://github.com/cultureamp/kaizen-design-system/pull/1753

There's really no good way to have the Tile front/back flip interaction be good for screen readers. What we can do to make this better is just pretend that it doesn't exist at all (using `aria-hidden`), and then append the information that appears on the back to the "front" of the card, and then _visually hide_ that so that only assistive tech sees it.

**Comment by sentience** [comment]
Why duplicate the content on the back of the card for screen readers _and_ hide the back of the card from screen readers? Is it not possible to let the screen readers read the back of the card (even though it isn't displayed visually until the card is flipped?).

**Comment by sentience** [comment]
Would an expand/collapse-type interaction work for screen readers? As in, maybe flipping the card "expands" the back-of-card content for screen readers, which is collapsed/hidden by default, while leaving the front-of-card content where it is even though it's no longer visually displayed?

Maybe even a use case for an [HTML `details` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details)?

**Comment by dougmacknz** [comment]
Here's my take on the current interaction with a screen reader that lead me to say that this interaction is no good:
* (Browsing previous content) 
* Button: label "Information" (or "Back" as it might be about to change to)
* Huh, what is this information for? Navigate past
* Heading 3: "(title)"
* (Browse the tile content)
* Navigate back and press "Information"
* What happened? Start navigating through again I guess
* Oh the content changed

A much more simple interaction would be:
* (Browsing previous content)
* Heading 3 "(title)"
* (Browse the card content)
* (Browse the "back of the content")

But I like your idea of bringing a disclosure in for the last part of ☝️ to signal that the back of the tile is less important information 👍
